### DISCUSSION TITLE
Targeted .Net Frameworks

### DISCUSSION THREAD

**Discussion by blmiles** [CommunityToolkit/Maui#1375] [General]
With long-term support for .Net6 and very short support for .Net7, WHY is this toolkit targeting .Net7?
And what is the last version/release that targeted .Net6?

I have all my platforms targeting .Net6 at the moment because of long-term support
If anyone can persuade me otherwise, I'd love to hear!

Excuse my ignorance but these incompatibilities are very frustrating!

**Comment by TheCodeTraveler** [comment]
Us maintainers are volunteers who contribute work to this library in our spare time on nights and weekends. We don't work for Microsoft and we don't get paid to work on this library. 

We don't have the capacity to backport support to .NET 6. 

If you'd like to implement .NET 6 support and join the team to maintain it until EOL, let me know!

**Reply by TheCodeTraveler** [reply]
For the curious, .NET 6 and .NET 7 reach End of Life only a few months apart. 

Updating from .NET 6 to .NET 7 is trivial. It is not really worth it to skip a release of .NET as upgrading from .NET 6 to .NET 8, for example, will be much more difficult than upgrading from .NET 6 to .NET 7 then again from .NET 7 to .NET 8. 

https://versionsof.net/core/
![image](https://github.com/CommunityToolkit/Maui/assets/13558917/c86aef7f-1bba-43ed-85e0-be00d6a0a479)

**Comment by VladislavAntonyuk** [comment]
in a few months, we will target .NET 8, that is LTS. Prepare your projects for this update)
